### ISSUE TITLE
BoltDB Archived, Switch to CoreOS's bbolt

### ISSUE THREAD

**Issue by theGeekPirate** [vilterp/treesql#13]
Hi there, randomly came across your repo and noticed you were using [BoltDB](https://github.com/boltdb/bolt), which has been archived. CoreOS's fork [bbolt](https://github.com/coreos/bbolt) would be a better option since it seems to be the most active.

**Comment by whisper-bye** [comment]
maybe [badgerdb](https://github.com/dgraph-io/badger) is a better choice.

**Comment by theGeekPirate** [comment]
Yeah, I was mainly thinking about a drop-in replacement, but moving to badgerdb may be a better idea if possible.

**Comment by recoilme** [comment]
Nice project. i'm a developer of https://github.com/recoilme/pudge db
Maybe it will not bad choice too

**Comment by gedw99** [comment]
Use Badger. Its really getting mature now.
Its just got streaming and backup and raft support fro the dgraph guys.
Its all got for regression testing on all commits now.
they also make it easy to do indexes and secondary indexes.
